sklearn/ensemble/gradient_boosting.py

```diff
@@ -1419,11 +1466,26 @@ def fit(self, X, y, sample_weight=None, monitor=None):
             # init state
             self._init_state()
 
-            # fit initial model - FIXME make sample_weight optional
-            self.init_.fit(X, y, sample_weight)
+            # fit initial model and initialize raw predictions
+            if self.init == 'zero':
+                raw_predictions = np.zeros(shape=(X.shape[0], self.loss_.K),
```

Comment:
should be `self.init_`, right ?

Reply:
Both work since we set `self.init_ = self.init` in `_init_state`. I'll use `init_` for consistency